### DISCUSSION TITLE
Bug? Can't Update Collection Name

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1420] [Support] [answered]
When I save after updating a collection name (changing Spring to Summer), it reverts back to Spring. 

Screenshot: https://app.usebubbles.com/562QsP7hWKoqrXRaUrw7dP/comments-on-sareview-org

**Comment by gabestein** [comment, marked as answer]
Sorry, this is just really bad UI that we plan to fix soon.

Workaround is to change title, refresh, then change the other stuff and save.

See workaround/context here: https://github.com/pubpub/pubpub/discussions/1413
